### ISSUE TITLE
Running simple_server example on Windows

### ISSUE THREAD

**Issue by lucasvfventura** [huge-success/sanic#856]
After managing to install Sanic on Windows (no uvloop or ujson) I tried to run the simple_server.py example and got the following output:
> python examples/simple_server.py
Traceback (most recent call last):
  File "examples/simple_server.py", line 1, in <module>
    from sanic import Sanic
  File "{VENV_DIR}\lib\site-packages\sanic\__init__.py", line 1, in <module>
    from sanic.app import Sanic
  File "{VENV_DIR}\lib\site-packages\sanic\app.py", line 13, in <module>
    from sanic.config import Config, LOGGING
  File "{VENV_DIR}\lib\site-packages\sanic\config.py", line 3, in <module>
    import syslog
ModuleNotFoundError: No module named 'syslog'

I followed the instruction described here https://stackoverflow.com/questions/25537419/python-import-syslog-on-windows and I manage to run it. 

Shouldn't the log be implemented using the logging module?

**Comment by skewty** [comment]
I got around this by creating a dummy syslog module

In a Command Prompt:
```
cd <project root>
md syslog
echo "LOG_DAEMON=''" > syslog\__init__.py
```

Also don't forget to:
```
set SANIC_NO_UVLOOP=true
```

**Comment by skewty** [comment]
In order to share the port with websocket connections I used this monkey patch:

```python
import asyncio

__all__ = ['do_monkey_patch']


def do_monkey_patch():
    try:
        getattr(asyncio.BaseEventLoop, 'create_server_original')
    except AttributeError:
        setattr(asyncio.BaseEventLoop, 'create_server_original', asyncio.BaseEventLoop.create_server)

        @asyncio.coroutine
        def _new_create_server(self, *args, **kwargs):
            kwargs['reuse_address'] = True
            return self.create_server_original(*args, **kwargs)

        setattr(asyncio.BaseEventLoop, 'create_server', _new_create_server)
```

**Comment by sjsadowski** [comment]
Closed, should have been resolved in 0.7.0 (per #1027)
